src/ng/browser.js

```diff
@@ -321,7 +321,7 @@ function Browser(window, document, $log, $sniffer) {
    */
   self.defer = function(fn, delay) {
     var timeoutId;
-    outstandingRequestCount++;
+    self.$$incOutstandingRequestCount();
```

Comment:
I don't think this buys us much and is inconsistent with the rest of the file (which either accesses/modifies `outstandRequestCount` directly or through (the private) `completeOutstandingRequest()` function).